### PR TITLE
xmlsec-1.2.38: Fix OpenSSL engine headers install on Windows

### PR DESCRIPTION
**Pull request by gureedo**
Specify library name and version:  **xmlsec/1.2.38**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by RubenRBS** [comment]
Hi! Can I get some logs of the fixed bug, and how this addresses it. Has this been reported upstream?

**Comment by gureedo** [comment]
It was just a bug in install script.
https://github.com/lsh123/xmlsec/pull/736

**Review by ericLemanissier — CHANGES_REQUESTED** [review]
scratch that, I read too fast !

**Review by ericLemanissier — APPROVED** [review]

File: recipes/xmlsec/all/patches/1.2.38-fix-openssl-engine-headers-install-on-windows.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+diff --git a/win32/Makefile.msvc b/win32/Makefile.msvc
+index 7eeb4ce0..27c62602 100755
+--- a/win32/Makefile.msvc
++++ b/win32/Makefile.msvc
+@@ -642,9 +642,9 @@ install : all
+ 	if not exist $(LIBPREFIX) mkdir $(LIBPREFIX)
+ 
+ 	:: include files
+-	if exist $(BASEDIR)\include\$(XMLSEC_NAME) copy $(BASEDIR)\include\$(XMLSEC_NAME)\*.h $(INCPREFIX)\$(XMLSEC_NAME)	
++	if exist $(BASEDIR)\include\$(XMLSEC_NAME) copy $(BASEDIR)\include\$(XMLSEC_NAME)\*.h $(INCPREFIX)\$(XMLSEC_NAME)
```
Comment (valgur) [inline recipes/xmlsec/all/patches/1.2.38-fix-openssl-engine-headers-install-on-windows.patch:19]:
```suggestion
 	if exist $(BASEDIR)\include\$(XMLSEC_NAME) copy $(BASEDIR)\include\$(XMLSEC_NAME)\*.h $(INCPREFIX)\$(XMLSEC_NAME)	
```
Just a suggestion to keep the patch slightly clearer.

**Review by valgur — APPROVED** [review]

**Pull request closed without merging**